### DISCUSSION TITLE
SDL3 Limits osu!lazer to monitor refresh rate

### DISCUSSION THREAD

**Discussion by fxzzi** [ppy/osu#30845] [Q&A] [answered]
Enabling sdl3 with the envvar `OSU_SDL3=1` causes my in game refresh rate to be locked to 170, despite having the max frame rate set to 2x refresh rate (340fps). I have ensured that tearing is enabled in my wayland compositor, and that the game is running in native wayland. I am running the game like so:

`OSU_SDL3=1 PIPEWIRE_LATENCY=64/44100 SDL_VIDEODRIVER=wayland osu-lazer`

Does anyone experience this? Anything I can do to solve it? I can feel the difference between sdl2 340fps and sdl3 170fps.

**Comment by Stoppedpuma** [comment, marked as answer]
Likely an issue with Hyprland (assuming you're using this still based off your previous discussion). The framerate remains uncapped on both Sway and River on my system on both SDL2 and SDL3. Another possibility is it being an nvidia issue in the case you're running an nvidia gpu?

**Reply by fxzzi** [reply]
yeah its a hyprland issue it seems, further discussion here: https://github.com/hyprwm/Hyprland/issues/8482
